Add documentation for OTLP Endpoint (#1060)
grafana/alloy: docs/sources/tasks/opentelemetry-to-lgtm-stack.md

Metrics, Logs, Traces: OTLP Receiver → batch processor → OTLP Exporter
```

## Grafana Cloud

Grafana Cloud provides OTLP Endpoints that you can use directly from within {{< param "PRODUCT_NAME" >}}.

You can find the OTLP connection details from the OpenTelemetry **Details** page in the [Grafana Cloud Portal][].

> You will need to update the configuration file as follows:

Reviewer: ```suggestion
You must update the configuration file as follows:
```

Metrics, Logs, Traces: OTLP Receiver → batch processor → OTLP Exporter
```

## Grafana Cloud

Grafana Cloud provides OTLP Endpoints that you can use directly from within {{< param "PRODUCT_NAME" >}}.

You can find the OTLP connection details from the OpenTelemetry **Details** page in the [Grafana Cloud Portal][].

You must update the configuration file as follows:

```
otelcol.auth.basic "default" {
  username = "<ACCOUNT ID>"
  password = "<API TOKEN>"
}

otelcol.exporter.otlphttp "default" {
  client {
    endpoint = "<OTLP_ENDPOINT>"
    auth     = otelcol.auth.basic.default.handler
  }
}
```

Replace the following:

* _`<ACCOUNT ID>`_: Your Grafana Cloud account ID.
* _`<API TOKEN>`_: Your Grafana Cloud API token.
* _`<OTLP_ENDPOINT>`_: Your OTLP endpoint.

> This will use the credentials stored in `otelcol.auth.basic "default"` to authenticate against the Grafana Cloud OTLP endpoints and you should start
> to see your data arrive!

Reviewer: ```suggestion

This configuration will use the credentials stored in `otelcol.auth.basic "default"` to authenticate against the Grafana Cloud OTLP endpoints, and you should start to see your data arrive!
```

One more small suggestion.
